traversability_estimation/include/traversability_estimation/TraversabilityMap.hpp

```diff
@@ -129,6 +133,23 @@ class TraversabilityMap
    */
   std::string getMapFrameId() const;
 
+  /*!
+   * Gets the default traversability value of unknown regions in the map.
+   * @return default traversability value of unknown regions in the map
+   */
+  double getDefaultTraversabilityUnknownRegions() const;
+
+  /*!
+   * Sets the default traversability value of unknown regions in the map.
+   * @param[in] defaultTraversability new default traversability value of unknown regions in the map
+    */
+  void setDefaultTraversabilityUknownRegions(const double& defaultTraversability);
```

Comment:
Typo (sorry, I get notified by all PRs where I am maintainer from the ANYbotics side ;) )

Reply:
I feel stupid, but I cannot spot the typo

Reply:
Missing "n" in "Unknown".